### PR TITLE
Update JWT Library

### LINKED ISSUE #4: need new example data.
The example token has expired.

### PR DESCRIPTION
**Pull request by mkasberg**
Makes this example compatible with the latest version of  [firebase/php-jwt](https://github.com/firebase/php-jwt).

Also fixes #4 by updating the expiration date in the JWT and adding the decrypted JWT to the committed code.

This PR includes several different commits - I can cherry-pick some of them if you want to keep parts of the changes but not others.

### CODE REVIEW

File: test.php [hunk 1]
```diff
@@ -1,14 +1,69 @@
...
+use \Firebase\JWT\JWT;
+
+$token = array (
```
Comment (joshcanhelp) [inline test.php]: Our fault including it in the first place but we should be encoding dummy data into the token. Maybe just a simple token like:

```php
$token_payload = [
  'sub' => '123456789',
  'email' : 'info@auth0.com'
];
```

File: test.php [hunk 2]
```diff
@@ -1,14 +1,69 @@
...
+  'starred_url' => 'https://api.github.com/users/jfromaniello/starred{/owner}{/repo}',
+  'subscriptions_url' => 'https://api.github.com/users/jfromaniello/subscriptions',
+  'type' => 'User',
+  'updated_at' => '2013-10-24T14:56:41Z',
+  'url' => 'https://api.github.com/users/jfromaniello',
+  'user_id' => 'github|178512',
+);
 
 // This is your client secret
 $key = '5yy6vCe0ChxadKGsVcX47VMCNZLBwWVrBLQdWeFJZ0_S2fLFi2o9wifuzE-U3MRX';
```
Comment (joshcanhelp) [inline test.php]: Use a dummy secret like:

```php
$key = '__test_secret__';
```

**Review by joshcanhelp — CHANGES_REQUESTED** [review]
Thanks @mkasberg, big improvement here!

File: test.php (hunk 1 above)
Comment (mkasberg) [reply test.php]: I was curious about that...

I'll amend the commit to scrub the non-dummy data. :+1:

**Comment by mkasberg** [comment]
Updated! I amended the original commits so they no longer include the "real" data in the token.

**Review by joshcanhelp — APPROVED** [review]

**Comment by joshcanhelp** [comment]
Looks great @mkasberg, thank you for the contribution!

**Pull request merged**